runtime/polkadot/src/constants.rs

```diff
@@ -18,8 +18,8 @@
 pub mod currency {
 	use primitives::v0::Balance;
 
-	pub const DOTS: Balance = 1_000_000_000_000;
-	pub const DOLLARS: Balance = DOTS / 100;       // 10_000_000_000
+	pub const UNITS: Balance = 1_000_000_000_000;
+	pub const DOLLARS: Balance = UNITS / 100;       // 10_000_000_000
```

Comment:
```suggestion
	pub const UNITS: Balance = 10_000_000_000;
	pub const DOLLARS: Balance = UNITS;       // 10_000_000_000
```

Wouldn't something like this make sense because of redenomination?